### ISSUE TITLE
[Client] Remove non strictly necessary layers from 'bblayers.conf'

### ISSUE THREAD

**Issue by indisoluble** [comodoo/meta-comodoo#20]
More exactly, I added `meta-qt5` because I needed only one of its recipes (I am not able to remember which one) and, if I remember clearly, I had to add `meta-openembedded/meta-gnome` to `bblayers.com` ... although it could be other layer.

Anyway, it is not right to add an extra dependency because of this reason, instead of that use `BBMASK` to take from `meta-qt5` only what I really need and ignore all the rest.

**Comment by indisoluble** [comment]
Layers below are not required:

```
    /path/to/yocto/meta-openembedded/meta-ruby \
    /path/to/yocto/meta-qt5 \
```

**Comment by indisoluble** [comment]
Fixed in [commit dc1872c](https://github.com/comodoo/meta-comodoo/commit/dc1872c0f0c75959a5d76fcae873a430659a5df3)
